### PR TITLE
Addition of BBright RMD

### PR DESCRIPTION
**Pull request by GeovannyGA**

### CODE REVIEW

File: connector/doc/BBright_RMD.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+HTTP CONNECTION:
+
+- **IP address/host**: The polling IP or URL of the destination.
+- **IP port**: Default: *80*.
+- **Bus address**: Default: *bypassproxy.*
+
+### Initialization
+
+The API requires the use of a token for authentication. You can get an API token via GUI under the page ADMIN > Server and click on 'REST API TOKEN'.
```
Comment (MariekeGO) [inline connector/doc/BBright_RMD.md:45]: By "GUI", do you mean the web interface of the device?

File: connector/doc/BBright_RMD.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Copy this token to the **Authentication Token** parameter on the **General** page of the DataMiner element.
+
+### Web Interface
+
+The web interface is only accessible when the client machine has network access to the product.
+
+## How to Use
+
+On the General page, the token used for the communication should be specified.
```
Comment (MariekeGO) [inline connector/doc/BBright_RMD.md:55]: Is there anything else we can add about how this connector can be used once the initialization is complete? What data does it show? Is there anything else that can be configured?

File: connector/doc/BBright_RMD.md (hunk 1 above)
Comment (GeovannyGA) [reply connector/doc/BBright_RMD.md]: yes. I added the word "web" in my new push.

File: connector/doc/BBright_RMD.md (hunk 2 above)
Comment (GeovannyGA) [reply connector/doc/BBright_RMD.md:55]: I push a new commit with more info.

File: connector/doc/BBright_RMD.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+The connector does not have any sets implemented. Currently these are the available commands:
```
Comment (MariekeGO) [inline connector/doc/BBright_RMD.md:57]: This seems a bit odd: first we say there are no sets implemented, but then we give a list of available commands. Are these commands that still need to be implemented, or are these actually things you can do with the connector?

Comment (GeovannyGA) [reply connector/doc/BBright_RMD.md:57]: By sets I'm referring to edits to the channel's configuration, this is currently not implemented in the connector. The list of available commands are the ones that are currently present in the connector, which is what is possible to do in this initial version.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**